full/src/test/java/apoc/trigger/TriggerClusterTest.java

```diff
@@ -104,4 +111,104 @@ public void testTxIdAfterAsync() throws Exception {
         org.neo4j.test.assertion.Assert.assertEventually(() -> TestContainerUtil.<Long>singleResultFirstColumn(cluster.getSession(), "MATCH p = ()-[r:GENERATED]->() RETURN count(p) AS count"),
                 (value) -> value == 2L, 30, TimeUnit.SECONDS);
     }
+
+    //
+    // test cases duplicated, regarding new procedures
+    //
+
+    @Test
+    public void testTimeStampTriggerForUpdatedPropertiesNewProcedures() throws Exception {
+        final String name = "timestampUpdate";
+        try (final Session session = cluster.getDriver().session(forDatabase(SYSTEM_DATABASE_NAME))) {
+            session.run("CALL apoc.trigger.install('neo4j', $name,'UNWIND apoc.trigger.nodesByLabel($assignedNodeProperties,null) AS n SET n.ts = timestamp()',{})",
+                    Map.of("name", name));    
+        }
+        try (final Session session = cluster.getSession()) {
+            awaitProcedureInstalled(session, "timestampUpdate");
+            session.run("CREATE (f:Foo) SET f.foo='bar'");
+            TestContainerUtil.testCall(session, "MATCH (f:Foo) RETURN f", (row) -> {
+                assertTrue(((Node) row.get("f")).containsKey("ts"));
+            });
+        }
+    }
+
+    @Test
+    public void testReplicationNewProcedures() throws Exception {
+        try (final Session session = cluster.getDriver().session(forDatabase(SYSTEM_DATABASE_NAME))) {
+            session.run("CALL apoc.trigger.install('neo4j', 'timestamp','UNWIND apoc.trigger.nodesByLabel($assignedNodeProperties,null) AS n SET n.ts = timestamp()',{})");
+        }
+        // Test that the trigger is present in another instance
+        awaitProcedureInstalled(cluster.getDriver().session(), "timestamp");
```

Comment:
How is this testing another instance?

Reply:
Because this method calls a `session.readTransaction(..)`, which should send the query to one of the nodes with the `READ` role (follower or readReplica), 
unlike `session.run(apoc.trigger.install ...)` which sends the query in a LEADER member.

The same thing that is done [on line 70](https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3183/files#diff-5e26a8edc16341c68758787654f5f129cd26b3f23cc8733fdcecb6cf32eb252cR70)